README.md

```diff
@@ -189,6 +189,13 @@ The library is available on NuGet package available. You can install it using th
      <th>Version</th>
      <th>Changes</th>
   </tr>
+  <tr>
+     <td>2018-04-13</td>
+     <td>?</td>
```

Comment:
The version should be 2.0.0. To actually have it be that version, you also need to update the assembly info: https://github.com/ErikSchierboom/JsonDotNetCustomContractResolvers/blob/master/src/CustomContractResolvers/Properties/AssemblyInfo.cs